biomaj/bank.py

```diff
@@ -466,9 +466,11 @@ def save_session(self):
         if self.session.get('action') == 'update' and not self.session.get_status(Workflow.FLOW_OVER)\
                 and self.session.get('release'):
             release = self.session.get('release')
-            self.banks.update({'name': self.name},
-                              {'$push': {'pending': {'release': self.session.get('release'),
-                                                     'id': self.session._session['id']}}})
+            found = self.banks.find_one({'name': self.name, 'pending.release': release})
+            if found is None:
+                self.banks.update({'name': self.name},
+                                  {'$push': {'pending': {'release': self.session.get('release'),
```

Comment:
What if current dession is not the same ? We may jave a pending for a release, but with a different session. If you do not update, you will not refer to the latest session (but is it really a pb?)